app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 class User < ActiveRecord::Base
   has_many :gems_adoptions
   has_many :adoption_requests
+  has_many :gem_ownership_transfers
```

Comment:
What would happen is you do `user.gem_ownership_transfers`? Remember that `GemOwnershipTransfer` has two associations with `user`.